README.md

```diff
@@ -215,9 +211,9 @@ The backend is used to serve the frontend and as an API for getting files from t
 
 There are two simple ways to build the backend, for local and server use.
 
-The easiest option while developing is to just run `mvn install`. This will install some plugins and bundle the files into the Jar. This command will also copy the frontend into the Jar if ran the [build command](#building-the-frontend). While technically you could build the Jar without the frontend, there really isn’t a reason not to, apart from testing the API only. Copying the frontend will be close to instant after the first build, as long as the build folder doesn’t change, because the target folder is cached.
+The easiest option while developing is to just run `mvn install`. This will build the backend and bundle existing front-end files into the Jar if present. While technically you could build the Jar without the frontend, there really isn’t a reason to do so. If you run `mvn inststall` multiple times, the front-end will be copied only the first time because `mvn install` does not clean up the target directory.
 
-The other option is meant for a production setting. This is because Maven will also install Node.js and NPM for building the frontend. This way is compatible with the Jenkins automation server. The build can be started by running `mvn install -P frontend`.
+The other option to run `mvn install -P frontend`. This command builds both the backend and the front-end. It is only meant for production, because with this command Maven will also install Node.js and NPM.
 
```

Comment:
I am not sure that my text is correct. I only tried to improve the wording of the existing text.